### PR TITLE
initiial homework template

### PR DESCRIPTION
**Pull request by talexandru1987**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,40 @@
...
+THEN the game is over
+WHEN the game is over
+THEN I can save my initials and my score
+```
+
+## Mock-Up
+
+The following pictures show the quiz application components:
+
+![Start](./Assets/assets/img/start.png)
```
Comment (LeeFarnell) [inline README.md]: Think your images are linked wrong which is why they aren't working.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+          <li>2</li>
+          <li>3</li>
+          <li>4</li>
+        </ul>
+
+        <div class="question-section-decision">
+          <p>Correct! or Wrong!</p>
+        </div>
+      </section> -->
+
```
Comment (LeeFarnell) [inline index.html]: If code is not being used, please delete it rather than just comment it out. This looks quite messy,

File: index.html [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+              <a href="./highscore.html">
+                <input id="submit-btn" type="submit" value="Submit" />
+              </a>
+            </div>
+            <div>
+              <p class="submit-result" id="submit-result">result</p>
+            </div>
+          </div>
+        </form>
+      </section> -->
```
Comment (LeeFarnell) [inline index.html]: As above with commented out code.

File: README.md (hunk 1 above)
Comment (talexandru1987) [reply README.md]: done

File: index.html (hunk 2 above)
Comment (talexandru1987) [reply index.html]: Done

**Pull request merged**